extensions/emmet/extension.toml

```diff
@@ -0,0 +1,11 @@
+id = "emmet"
+name = "Emmet"
+description = "Emmet support"
+version = "0.0.1"
+schema_version = 1
+authors = []
+repository = "https://github.com/zed-extensions/emmet"
+
+[language_servers.emmet-language-server]
+name = "Emmet Language Server"
+language = "HTML"
```

Comment:
todo/to-figure-out: we most likely want this language server to work for multiple languages; most likely we'll have adjust the schema there (+ some languages like TSX should only use emmet LSP in html-ish subset).